hypha/static_src/src/javascript/apply/determination-template.js

```diff
@@ -20,14 +20,27 @@
         }
 
         getMatchingCopy(value) {
+            var proposal_form = document.querySelector('#id_proposal_form');
             if (value === '0') {
                 this.text = document.querySelector('div[data-type="rejected"]').textContent;
+                if (proposal_form) {
+                    proposal_form.disabled = true;
+                    proposal_form.required = false;
+                }
             }
             else if (value === '1') {
                 this.text = document.querySelector('div[data-type="more_info"]').textContent;
+                if (proposal_form) {
+                    proposal_form.disabled = true;
+                    proposal_form.required = false;
+                }
             }
             else {
                 this.text = document.querySelector('div[data-type="accepted"]').textContent;
+                if (proposal_form) {
+                    proposal_form.disabled = false;
+                    proposal_form.required = true;
+                }
             }
             this.updateTextArea(this.text);
         }
```

Comment:
I think the only jQuery is at the end, the `$(DeterminationCopy.selector()).each((index, el)` part.

If you have time, can you convert that to plain JS and remove all jQuery references from this file.